src/test/java/com/aws/greengrass/logmanager/model/ProcessingFileLRUTest.java

```diff
@@ -0,0 +1,71 @@
+package com.aws.greengrass.logmanager.model;
+
+import com.aws.greengrass.logmanager.LogManagerService;
+import com.aws.greengrass.testcommons.testutilities.GGExtension;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.io.TempDir;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@ExtendWith({MockitoExtension.class, GGExtension.class})
```

Comment:
I need to add a few more tests here and also change a few now that we are tracking multiple files